src/index.js

```diff
@@ -5,7 +5,7 @@ import {
 	NAMESPACE_REPLACE_REGEX,
 	HTML_LOWER_CASE,
 	HTML_ENUMERATED,
-	SVG_CAMEL_CASE,
+	SVG_KEBAB_CASE,
```

Comment:
Renamed for consistency.

`HTML_LOWER_CASE` is converting camel-cased names to lower-cased, therefore `SVG_CAMEL_CASE` stands out as it's converting camel-cased names to kebab-cased.